src/sql/workbench/contrib/query/browser/queryActions.ts

```diff
@@ -323,13 +323,53 @@ export class EstimatedQueryPlanAction extends QueryTaskbarAction {
 		}
 
 		if (this.isConnected(editor)) {
-			editor.input.runQuery(editor.getSelection(), {
-				displayEstimatedQueryPlan: true
-			});
+			let planOptions: ExecutionPlanOptions = editor?.input?.state?.isActualExecutionPlanMode ? { displayActualQueryPlan: true } : { displayEstimatedQueryPlan: true };
+			editor.input.runQuery(editor.getSelection(), planOptions);
 		}
 	}
 }
 
+/**
+ * Action class that toggles the actual execution plan mode for the editor
+ */
+export class ToggleActualExecutionPlanModeAction extends QueryTaskbarAction {
+	public static EnabledClass = 'estimatedQueryPlan';
```

Comment:
`estimatedExecutionPlan`

(ID too)

Reply:
I changed the name of the ID, but didn't change the value of `EnabledClass` since the value is also used by the `EstimatedQueryPlanAction` class too:
![image](https://user-images.githubusercontent.com/87730006/172915893-d1269873-7902-494b-98db-e120e0e4d659.png)

Reply:
Why are you having them share a class? They should generally have separate ones I believe.